### PR TITLE
cassandra 4.x migration guide

### LINKED ISSUE #7: Cassandra Client migration

### PR DESCRIPTION
**Pull request by Sammers21**
Closes #7

### CODE REVIEW

File: asciidoc/cassandra.adoc [hunk 1]
```diff
@@ -0,0 +1,65 @@
+=== Cassandra Client
+
+==== Underlying driver changes
+
+In Vert.x 4 Cassandra Client uses a new major 4.x version of Datastax Java driver, which
+is incompatible the previous one. Migration process might involve import changes and
```
Comment (tsegismont) [inline asciidoc/cassandra.adoc:6]:
```suggestion
is incompatible with the previous one. Migration process might involve import changes and 
```

File: asciidoc/cassandra.adoc [hunk 2]
```diff
@@ -0,0 +1,65 @@
+=== Cassandra Client
+
+==== Underlying driver changes
+
+In Vert.x 4 Cassandra Client uses a new major 4.x version of Datastax Java driver, which
```
Comment (tsegismont) [inline asciidoc/cassandra.adoc:5]:
```suggestion
In Vert.x 4 Cassandra Client uses the latest major version of Datastax Java driver (4.x), which
```

File: asciidoc/cassandra.adoc [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+SimpleStatement statement =
+  SimpleStatement.newInstance("SELECT release_version FROM system.local");
+cassandraClient.execute(statement, done -> {
+    ResultSet results = done.result();
+    Row row = resultSet.one();
+    System.out.println(row.getString("release_version"));
+});
+```
+
+==== CassandraClientOptions changes
```
Comment (tsegismont) [inline asciidoc/cassandra.adoc:40]:
```suggestion
==== `CassandraClientOptions` changes
```

File: asciidoc/cassandra.adoc [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+Construction of CassandraClientOptions now might require different imports:
```
Comment (tsegismont) [inline asciidoc/cassandra.adoc:42]:
```suggestion
Construction of `CassandraClientOptions` now might require different imports:
```

File: asciidoc/cassandra.adoc [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+// Vert.x Cassandra Client 4.x:
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.datastax.oss.driver.api.core.CqlSessionBuilder;
+import io.vertx.cassandra.CassandraClientOptions;
+
+new CassandraClientOptions(CqlSession.builder());
+```
+
+Methods `setPort`, `getContactPoints` and `setContactPoints` was removed. Instead,
+`addContactPoint` is expected to be used.
```
Comment (tsegismont) [inline asciidoc/cassandra.adoc:60]: Can you add examples for Vert.x 3 and 4 like you did above?

**Review by tsegismont — CHANGES_REQUESTED** [review]
Thanks @Sammers21 !

**Comment by Sammers21** [comment]
@tsegismont, can you review the PR once again, please?

**Review by tsegismont — APPROVED** [review]
Thank you @Sammers21 !

**Pull request merged**